scripts/tests/test_zkai_native_attention_mlp_rmsnorm_label_sensitivity_gate.py

```
@@ -0,0 +1,105 @@
+    def test_output_path_rejects_symlink(self) -> None:
+        with tempfile.TemporaryDirectory() as tmpdir:
+            tmp = pathlib.Path(tmpdir)
+            target = tmp / "target.json"
+            target.write_text("{}", encoding="utf-8")
+            link = gate.EVIDENCE_DIR / f"tmp-label-sensitivity-{target.name}"
+            try:
+                link.symlink_to(target)
+            except (OSError, NotImplementedError):
+                self.skipTest("symlink creation unavailable")
+            try:
+                with self.assertRaises(gate.RmsnormLabelSensitivityError):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)